README.md

```diff
@@ -1 +1,184 @@
-# To_do_list_project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./Logo_BeautifulBoys.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \*\*To Do List Project - Jose RM. ](#-to-do-list-project---jose-rm-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+ <br/>
+
+# 📖 **To Do List Project - Jose RM. <a name="about-project"></a>
+
+
+
+## 🛠 Built With <a name="Visual Studio Code"></a>
+
+### Tech Stack <a name="Viual Studio Code"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML]**
+- **[CSS]**
+- **[Javascript]**
```

Comment:
- [ ] I would kindly suggest you add this stack inside of the `tech stack` heading rather than putting it inside of the `key features heading. Currently the `tech stack` is missing the necessary content.
- [ ] Also, kindly describe in 1 or 2 sentences what the todo list project is all about.

Reply:
Ok, thanks for your time I will do the corrections.